conf.d/powerdns_recursor.yaml.example

```diff
@@ -17,6 +23,8 @@ instances:
     port: 8082
   # Recursor web server api key.
     api_key: pdns_api_key
+  # Version 3 or 4
+    version: 4
```

Comment:
Actually it can be completely absent and it will default to version 3. i.e. existing configurations would remain working after an upgrade of the agent.

I will update the configuration regardless to ensure that it is consistent with the default.